app/src/main/java/com/github/sdp/mediato/data/Database.java

```diff
@@ -21,6 +23,9 @@
 public class Database implements GenericDatabase {
 
     public static final String USER_PATH = "Users/";

```

Comment:
Would it be possible to only use the singular or plural for the variable names and not a mix of both?